server/app/crud/crud.py

```diff
@@ -78,4 +78,22 @@ def check_key_existence(key):
         if result is None:
             return False
         else:
-            return True
\ No newline at end of file
+            return True
+
+
+def get_latest_generated_key():
+    try:
+        connection = create_connection(db_file)
+        cursor = connection.cursor()
+        cursor.execute("""
```

Comment:
Is having """ statement """ correct? follow similar conventions "statement" like other places

Reply:
hi, @Tejas-ChandraShekarRaju 
Yes, this is copying something else.

Reply:
Why triple """ SELECT key FROM store ORDER BY id DESC LIMIT 1;""". "SELECT key FROM store ORDER BY id DESC LIMIT 1" should work right ? Let's follow the same as what we followed for other functions. Apart from the convention this code works.